exercises/exercise3.js

```diff
@@ -8,5 +8,145 @@ module.exports.botScripts = [
     handler: function () {
       return 'Good Evening to You!';
```

Comment:
Missing a closing curly brace for your function. The curly brace on line 10 is technically supposed to close your interaction/pair with the opening curly brace on line 5.